lib/tasks/mailer.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+namespace 'mailer' do
```

Comment:
we should make rake task for sending a test mail
for the daemon please use bin/ with gli:
http://davetron5000.github.io/gli/

Reply:
Implemented a tiny CLI for postmaster in `./bin/postmaster`